app/controllers/attendances_controller.rb

```diff
@@ -13,7 +13,7 @@ def destroy
   private
 
     def attendance
-      current_user.attendances.find(params[:id])
+      current_user.student_team.attendances.find(params[:id])
```

Comment:
In fact, I removed this controller. Now that we don't need to confirm preference anymore, this controller makes no sense.